conda_smithy/ci_register.py

```diff
@@ -149,7 +162,10 @@ def add_project_to_travis(user, project):
 
         if count > 5:
             print('  * Unable to register the repo on Travis (is it down?)')
```

Comment:
Perhaps just add ``'(is it down? is the {} slug good [note: case sensitive])?'.format(repo_slug)``